src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -59,8 +60,14 @@ class BaseOptionsSelector extends Component {
                 if (!focusedOption) {
                     return;
                 }
-
-                this.selectRow(focusedOption);
+                if (!this.state.shouldDisableRowSelection) {
```

Comment:
I just tested in flows where multiple select is available like 'New group' creation and it looks weird to block multiple presses there as well. I think we can execute the multi-enter press blocking logic only in case of single selection. We can use `this.props.canSelectMultipleOptions` props for this.